pkg/eval/math/math.go

```diff
@@ -0,0 +1,286 @@
+// Package math exposes functionality from Go's math package as an elvish
+// module.
+package math
+
+import (
+	"math"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var e
+//
+// ```elvish
+// $math:e
+// ```
+//
+// The value of E: 2.718281.... This variable is read-only.
+
+//elvdoc:var pi
+//
+// ```elvish
+// $math:pi
+// ```
+//
+// The value of Pi: 3.141592.... This variable is read-only.
+
+//elvdoc:fn abs
+//
+// ```elvish
+// math:abs $number
+// ```
+//
+// Compute the absolute value of a number. Example:
```

Comment:
Use third-person singular form of the verb, "computes".

In case you haven't noticed, there is now a [style guide](https://github.com/elves/elvish/blob/master/CONTRIBUTING.md#documenting-builtin-functions-and-variables) for the "elvdoc" comments.

Same for functions below.

Reply:
Okay, but how to square the circle? That is, what about the commands like `math:round` which say `Output the nearest integer` rather than `Outputs...`?

Reply:
Ah, never mind, I see you made another comment about s/Output/Outputs/.